content/en/docs/quick-start/deploy-bookinfo-to-k8s.md

```diff
@@ -20,63 +20,60 @@ To provide consistent user experiences of managing microservices, KubeSphere int
 
 - You need to enable **Application Governance**. To do so, perform the following steps:
 
-  Log in to the console as `project-admin` and go to your project. Navigate to **Advanced Settings** under **Project Settings**, click **Edit**, and select **Edit Gateway**. In the dialog that appears, flip on the toggle switch next to **Application Governance**.
+  Log in to the console as `project-admin` and go to your project. Go to **Advanced Settings** under **Project Settings**, click **Edit**, and select **Edit Gateway**. In the displayed dialog box, turn on the toggle switch next to **Application Governance**.
```

Comment:
The description here does not match the current UI. Please check it out.
<img width="1269" alt="Xnip2021-07-20_13-45-55" src="https://user-images.githubusercontent.com/75110798/126268106-ca45cb66-35fb-42b3-beb9-68827d9ecf8b.png">

Reply:
If a user has set a gateway before, then the layout of the page would match the description. I think we need to have a little discussion here.

Reply:
I updated the description here by adding a link to another document that has already described this operation. Please take a look, thanks!

Reply:
Cool.